src/args.rs

```diff
@@ -83,6 +83,12 @@ pub fn parse() -> ArgMatches<'static> {
                         .long("watch-when-idle")
                         .help("Ignore events emitted while the commands run"),
                 )
+                .arg(
+                    Arg::with_name("features")
+                        .long("features")
+                        .takes_value(true)
+                        .help("Space-separated list of features passed to cargo invocations"),
```

Comment:
Comma-separated, surely

Reply:
Cargo documents this argument as space-separated, but it seems the arguments can also be separated by a comma.

Reply:
Ah, hmm. I've only ever seen it with commas (and that's how the examples you wrote are haha!). Maybe we should just not specify and rely on familiarity.